llvm/lib/YkIR/YkIRWriter.cpp

```diff
@@ -388,6 +389,26 @@ class YkIRWriter {
     InstIdx++;
   }
 
+  void serialiseDeoptSafepointInst(CallInst *I, ValueLoweringMap &VLMap, unsigned BBIdx,
+                         unsigned &InstIdx) {
+      serialiseOpcode(OpCodeDeoptSafepoint);
+      // stackmap ID:
+      serialiseOperand(I, VLMap, I->getOperand(0));
+
```

Comment:
Maybe I'd remove the newlines between the fields you are serialising, but I won't die on that hill.